src/System.Runtime/tests/System/ArrayTests.cs

```diff
@@ -533,6 +550,29 @@ public static IEnumerable<object[]> Copy_TestData()
         }
 
         [Theory]
+        public static void ForEach()
+        {
+            Array.ForEach<int>(new int[] { }, new Action<int>(i => { throw new InvalidOperationException(); }));
+            // Did not throw; no items
+
+            int counter = 0;
+            Array.ForEach<int>(new int[] { 1, 2, 3 }, new Action<int>(i => counter += i));
+            Assert.Equal(6, counter);
+
+            // Only works on one dimensional arrays
```

Comment:
I hadn't looked at the signature; didn't realize it took a ```T[]``` instead of an ```Array```.  In that case, the comment doesn't make sense, as the signature simply doesn't allow anything else, and can just be removed.